ports/iir1/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "iir1",
+  "version-string": "1.8.0",
```

```suggestion
  "version": "1.8.0",
```
Please see the doc https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions to learn more about package versions.